src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -379,6 +379,11 @@ internal virtual string AppVersion
         /// </summary>
         public readonly PreferenceSettings PreferenceSettings;
 
+        /// <summary>
+        /// This list contains the settings (current status) of the expanders located in the Preferences panel
+        /// </summary>
+        public List<ExpanderSettings> ExpandersSettings;
```

Comment:
Why dont we store this in Preferences ViewModel since it does not directly affect DynamoModel?

Reply:
@QilongTang Same reason than the previous comment, the PreferencesViewModel is created until the Preferences window is opened so we won't be able to store/recover data for the expanders.